README.md

```diff
@@ -1,41 +1,57 @@
-# geo-arrow-spec
+# GeoArrow - geospatial data with Apache Arrow
 
 Specifications for storing geospatial data in Apache Arrow and Apache Parquet.
```

Comment:
I'd probably remove `Parquet` from the first sentence, given that that might confuse people already familiar with GeoParquet. What do you think